Gemfile

```diff
@@ -74,7 +74,6 @@ gem 'angularjs-file-upload-rails', '~> 2.4.1'
 gem 'blockenspiel'
 gem 'custom_error_message', github: 'jeremydurham/custom-err-msg'
 gem 'dalli'
-gem 'deface'
```

Comment:
I kind of forgot about it (that's what our brain does to get over traumas, right? :trollface: ) but it was such a pain...